### DISCUSSION TITLE
Heeelppp Please

### DISCUSSION THREAD

**Discussion by stubird107** [dompdf/dompdf#2642] [General]
hi all 

im quite new to the programming world and have just started out so sorry if i dont explain this correctly or have posted in the wrong place, i am using dompdf to create pallet tickets ,when the file has been generated they are printed and applied to pallets

what i am trying to acheive is that i get the number of pages relating to how many pallets where booked in so if 

pall_qty = 3 , i would like 3 of the same page inside the pdf

i was achieving  this before but for some reason after 3 months it stopped working on my hosting provider. i have reubilt a script
to generate 1 page but i would like to generate copies of the page depdning on the pallet qty, 

The script

[creategrn.txt](https://github.com/dompdf/dompdf/files/7533657/creategrn.txt)

**Comment by bsweeney** [comment]
Would you just wrap the output generation logic inside a loop? You can force a page break for everything but the first instance. So add the following styling:

```
.container {
  page-break-before: always;
}
.container:first-of-type {
  page-break-before: auto;
}
```

Then generate you content doing something like this:

```
for () {
  $output .= '
    <div class="container" align="center" style="font-size:8pt;"><!--snip--></div>
  ';
}
```
